test/Passes/pack-unpack-propagation.mlir

```diff
@@ -1,5 +1,4 @@
-// RUN: tpp-opt %s -propagate-pack-and-unpack -canonicalize -split-input-file | FileCheck -check-prefix=MATMUL %s
-// RUN: tpp-opt %s -propagate-pack-and-unpack -canonicalize -split-input-file | FileCheck -check-prefix=CONV %s
+// RUN: tpp-opt %s -propagate-pack-and-unpack -canonicalize -split-input-file | FileCheck -check-prefix=CHECK %s
```

Comment:
You don't need `-check-prefix=CHECK`, this is the default.

Reply:
Ops, left over.